crates/node/core/src/args/payload_builder.rs

```diff
@@ -20,6 +20,10 @@ pub struct PayloadBuilderArgs {
     #[arg(long = "builder.gaslimit", value_name = "GAS_LIMIT")]
     pub gas_limit: Option<u64>,
 
+    /// Target execution time for block building.
+    #[arg(long = "builder.executionlimit", value_parser = parse_duration_from_secs_or_ms, value_name = "EXECUTION_LIMIT")]
+    pub execution_limit: Option<Duration>,
```

Comment:
Should this be part of the core reth cli or should we have it in the rollup-node cli instead and then inject it here:
https://github.com/scroll-tech/reth/blob/4d7529e36b0c51266412828171db5ee5ca72da16/crates/scroll/node/src/node.rs#L44